src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -163,7 +163,7 @@ public void updateFilteredPersonList(Predicate<Person> predicate) {
         @Override
         public boolean hasPerson(Person person) {
```

Comment:
Perhaps the method name can be renamed to `hasClient`? This applies to a few more other similar methods.